src/StationSelect.js

```diff
@@ -0,0 +1,23 @@
+import Select from 'react-select'
+import { customStyles } from './selectStyling';
+
+
+function StationSelect({ data, stationSelect }) {
+
+    const stationData = data.stations;
+    const stationNames = stationData.map((station) => ({ value: `${station.id}`, label: `${station.name}` }));
+
+    const handleSelect = (selected) => {
+        stationSelect(selected.value)
+    }
+
+    return (
+        <div className='stationAndRegionSelect'>
+            <label>Choose a Station:</label>
+            <Select onChange={handleSelect} options={stationNames} styles={customStyles} />
```

Comment:
this is kind of out of scope i suppose, but since you've got an app that runs internationally there's going to be language barriers - first thing i tried was clicking on velobike, and i got a bunch of cyrillic, which makes sense since im assuming this just pulls from an API. however i imagine this could be UX issue for those who don't know whatever languages are being displayed on the screen. not sure how to implement this but something to note

Reply:
Good suggestion. Will think about how to approach